druid.go

```diff
@@ -72,9 +80,10 @@ func NewClient(baseURL string, options ...ClientOption) (*Client, error) {
 			RetryWaitMax: opts.retryWaitMax,
 			RetryMax:     opts.retryMax,
 		},
-		username:  opts.username,
-		password:  opts.password,
-		basicAuth: opts.username != "" && opts.password != "",
+		username:      opts.username,
+		password:      opts.password,
+		basicAuth:     opts.username != "" && opts.password != "",
+		skipTLSVerify: opts.skipTLSVerify,
```

Comment:
Not sure what is it used for. If it's to know the TLSClientConfig configuration then one could access it thru `http.HTTPClient.Transport.TLSClientConfig.InsecureSkipVerify`

Reply:
Aah yes, you did mention that.

Reply:
agreed to remove it then ?